includes/elements/container.php

```diff
@@ -429,6 +444,8 @@ protected function register_container_layout_controls() {
 	 * @return void
 	 */
 	protected function register_items_layout_controls() {
+		$wrapper_include_inner = '{{WRAPPER}}, {{WRAPPER}} > .e-container-box__inner';
```

Comment:
where do you use it?

Reply:
@StyleShit 

This is unnecessary indeed. I will restore the original version.